core/src/main/res/values-night/color.xml

```diff
@@ -5,16 +5,7 @@
   <!--The same color values are defined for dark theme, but with values more appropriate for -->
   <!--low-luminance UIs. These colors will be used by night/themes.xml.-->
 
-  <color name="color_primary">@color/blue400</color>
-
-  <color name="color_surface">@color/gray900</color>
-  <color name="color_error">@color/pink300</color>
-
-  <color name="color_on_primary">@color/white</color>
-  <color name="color_on_surface">@color/white</color>
-  <color name="color_on_error">@color/white</color>
-
-  <color name="text_secondary">@color/gray350</color>
-  <color name="text_tertiary">@color/gray500</color>
-  <color name="card_view_background">@color/gray850</color>
+  <color name="text_secondary">@color/mine_shaft_gray350</color>
```

Comment:
I want to avoid any and all color overriding if we want a layer of indirection we can add our own attribute to our themes and change it there.

Add an attribute in `attrs.xml` `text_secondary` and then define the differing colors in night/KiwixTheme and regular KiwixTheme and replace usages with of the resource with the attr.

Rinse and repeat for tertiary and background